### ISSUE TITLE
test_s3_benchmark_hsbench TC failed in Azure for RGW pod

### ISSUE THREAD

**Issue by ramkiperiy** [red-hat-storage/ocs-ci#3917]
TC test_s3_benchmark_hsbench failed in Azure due to unexpected behaviour, this TC needs enhancement for Azure.

```
___________________________________________________________________ TestHsBench.test_s3_benchmark_hsbench ___________________________________________________________________

self = <test_hsbench.TestHsBench object at 0x7f5a72c93460>, hsbenchs3 = <ocs_ci.ocs.hsbench.HsBench object at 0x7f56c724eb50>

    def test_s3_benchmark_hsbench(self, hsbenchs3):
        """
        Test case to test one million objects in a single bucket:
        * Create RGW user
        * Create test pod
        * Install hs S3 benchmark
        * Run hs S3 benchmark to create 1M objects
        """
        # Create RGW user
        hsbenchs3.create_test_user()
    
        # Create resource for hsbench
>       hsbenchs3.create_resource_hsbench()

tests/e2e/scale/noobaa/test_hsbench.py:41:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <ocs_ci.ocs.hsbench.HsBench object at 0x7f56c724eb50>

    def create_resource_hsbench(self):
        """
        Create resource for hsbench mark test:
            Create service account
            Create PVC
            Create golang pod
    
        """
        # Check for existing rgw pods on cluster
        self.rgw_pod = pod.get_rgw_pods()
        if self.rgw_pod:
            # Create service account
            self.sa_name = helpers.create_serviceaccount(self.namespace)
            self.sa_name = self.sa_name.name
            helpers.add_scc_policy(sa_name=self.sa_name, namespace=self.namespace)
    
            # Create test pvc+pod

            Create PVC
            Create golang pod
    
        """
        # Check for existing rgw pods on cluster
        self.rgw_pod = pod.get_rgw_pods()
        if self.rgw_pod:
            # Create service account
            self.sa_name = helpers.create_serviceaccount(self.namespace)
            self.sa_name = self.sa_name.name
            helpers.add_scc_policy(sa_name=self.sa_name, namespace=self.namespace)
    
            # Create test pvc+pod
            log.info(f"Create Golang pod to generate S3 workload... {self.namespace}")
            pvc_size = "50Gi"
            node_name = "compute-0"
            self.pod_name = "hsbench-pod"
            self.pvc_obj = helpers.create_pvc(
                sc_name=constants.DEFAULT_STORAGECLASS_RBD,
                namespace=self.namespace,
                size=pvc_size,
            )
            self.pod_obj = helpers.create_pod(
                constants.CEPHBLOCKPOOL,
                namespace=self.namespace,
                pod_name=self.pod_name,
                pvc_name=self.pvc_obj.name,
                node_name=node_name,
                sa_name=self.sa_name,
                pod_dict_path=self.pod_dic_path,
                dc_deployment=True,
                deploy_pod_status=constants.STATUS_COMPLETED,
            )
        else:
>           raise UnexpectedBehaviour(
                "This cluster doesn't have RGW pod(s) to perform hsbench"
            )
E           ocs_ci.ocs.exceptions.UnexpectedBehaviour: This cluster doesn't have RGW pod(s) to perform hsbench

ocs_ci/ocs/hsbench.py:80: UnexpectedBehaviour
```

**Comment by tiffanyn108** [comment]
PR submitted for review: https://github.com/red-hat-storage/ocs-ci/pull/3923

**Comment by tiffanyn108** [comment]
It's fixed in #3923.  Closing this PR.
